rector.php

```diff
@@ -5,28 +5,20 @@
 use Rector\CodeQuality\Rector\Class_\InlineConstructorDefaultToPropertyRector;
 use Rector\Config\RectorConfig;
 use Rector\Php74\Rector\Closure\ClosureToArrowFunctionRector;
-use Rector\Set\ValueObject\LevelSetList;
+use Rector\Php81\Rector\FuncCall\NullToStrictStringFuncCallArgRector;
+use Rector\Php81\Rector\Property\ReadOnlyPropertyRector;
 
-return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->paths([
+return RectorConfig::configure()
+    ->withPaths([
         __DIR__ . '/src',
         __DIR__ . '/tests',
-    ]);
-
-    $rectorConfig->skip([
-        __DIR__ . '/tests/Support/Grpc/GPBMetadata',
-        __DIR__ . '/tests/Support/Grpc/Service',
-    ]);
-
-    // register a single rule
-    $rectorConfig->rule(InlineConstructorDefaultToPropertyRector::class);
-
-    // define sets of rules
-    $rectorConfig->sets([
-        LevelSetList::UP_TO_PHP_80,
-    ]);
-
-    $rectorConfig->skip([
+    ])
+    ->withPhpSets(php81: true)
```

Comment:
I'm suggest update rector and apply his suggestions in separate PR.

Reply:
Ok

Reply:
@vjik all done.